src/models.jl

```diff
@@ -92,3 +92,29 @@ function initialize_with_defaults!(eos, tracers, sets...)
         end
     end
 end
+
+"""
+    add_sponge_layer!(model; damping_timescale)
```

Comment:
The purpose of a sponge layer is damp all energy within the region of the sponge --- not just waves. 

I wouldn't characterize the waves as unphysical, because they aren't associated with numerical error or model error. The waves are generated and reflected off boundaries *because* of the physics of the Boussinesq equations, and are thus very much 'physical'. It is the mathematical definition of the problem that the sponge layer seeks to change, rather than some 'correction' to Boussinesq physics.

Reply:
Thanks for the explanation. I guess I meant unrealistic instead of unphysical. Will update the docstring when this is merged into `sandbox/`.